tests/e2e/utils/visit-admin-page-with-hash.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { join } from 'path';
+
+/**
+ * WordPress dependencies
+ */
+import { getPageError } from '@wordpress/e2e-test-utils';
+
+function createURLWithHash( WPPath, query = '', hash = '' ) {
+	const url = new URL( 'http://localhost:8890' );
```

Comment:
This URL should be available via the environment variable `WP_BASE_URL`, so no need to hard code it.

Reply:
Updated in 92ec1f3e6